docs/api/connectors.md

```diff
@@ -86,3 +86,14 @@ class LocalConnector(BaseConnector):
             bool: indicating success or failure.
         """
 ```
+
+## pyproject.toml
+
+In order for pyinfra to gain knowledge about your connector, you need to add the following snippet to your connector's `pyproject.toml`:
+
+```toml
+[project.entry-points.'pyinfra.connectors']
```

Comment:
Amazing, long been on the list to figure out what the correct config was here!

Reply:
Well, your hint in https://github.com/pyinfra-dev/pyinfra/issues/977#issuecomment-1551447809 was quite good :)